ui/app/AppLayouts/Communities/views/EditPermissionView.qml

```diff
@@ -276,6 +276,7 @@ StatusScrollView {
                                         root.assetsModel, key)
 
                     addItem(Constants.TokenType.ERC20, modelItem, String(AmountsArithmetic.fromNumber(amount, modelItem.decimals)))
+                    addItem(Constants.TokenType.ERC20, modelItem, AmountsArithmetic.fromNumber(amount, modelItem.decimals).toFixed())
```

Comment:
Wait... shouldn't this replace the line above?

Reply:
wtf, how did that happen 😄

Reply:
Glad you hadn't submitted this PR yesterday :D

Reply:
Fixed, thank you!